src/.pylintrc-mandatory.jinja

```diff
@@ -6,12 +6,21 @@ load-plugins=pylint_odoo
 score=n
 
 [ODOOLINT]
+{%- if odoo_version < 16 %}
 readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
 manifest_required_authors={{ org_name }}
 manifest_required_keys=license
 manifest_deprecated_keys=description,active
 license_allowed=AGPL-3,GPL-2,GPL-2 or any later version,GPL-3,GPL-3 or any later version,LGPL-3
 valid_odoo_versions={{ odoo_version }}
+{%- else %}
+readme-template-url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
+manifest-required-authors={{ org_name }}
+manifest-required-keys=license
+manifest-deprecated-keys=description,active
+license-allowed=AGPL-3,GPL-2,GPL-2 or any later version,GPL-3,GPL-3 or any later version,LGPL-3
+valid-odoo-versions={{ odoo_version }}
+{%- endif %}
```

Comment:
This solution sounds a bit awkward to me. Summoning @moylop260 here: is this the right way to do this? Did pylint-odoo 8 actually change the option names?

Reply:
Based on https://github.com/OCA/pylint-odoo/issues/465 thats what I have understood from moylop260 and observed :) If I've made a huge mistake apologies :)

As for the fix I copied the version checking pattern that was being used elsewhere. For changing the tests I explicitly checked the pylint-odoo version. I did toy with doing that, but elected to try and follow the same pattern. I don't mind revisiting this if there's a preferred change.

Reply:
@theangryangel thanks for the explanation (which I should have found myself, had I read your OP carefully :)

Reply:
We're all busy people :) It's easily done, anything I can do to try and make life easier tho :)